### PR TITLE
move distribution artifacts to distribution/target

### PR DESCRIPTION
**Pull request by guobingkun**
move distribution artifacts to distribution/target

### CODE REVIEW

**Comment by guobingkun** [comment]
```Running io.druid.query.extraction.namespace.TestKafkaExtractionCluster
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 8.404 sec <<< FAILURE! - in io.druid.query.extraction.namespace.TestKafkaExtractionCluster```

**Comment by drcrallen** [comment]
cool :+1:

File: distribution/pom.xml [hunk 1]
```diff
@@ -57,6 +57,8 @@
                                 <argument>-classpath</argument>
                                 <classpath/>
                                 <argument>-Ddruid.extensions.loadList=[]</argument>
+                                <argument>-Ddruid.extensions.directory=target/druid_extensions</argument>
```
Comment (xvrl) [inline distribution/pom.xml]: target should be `${project.build.directory}`

File: distribution/pom.xml [hunk 2]
```diff
@@ -57,6 +57,8 @@
...
+                                <argument>-Ddruid.extensions.hadoopDependenciesDir=target/hadoop_druid_dependencies</argument>
```
Comment (xvrl) [inline distribution/pom.xml]: same here, use `${project.build.directory}/hadoop_druid_dependencies`

File: distribution/src/assembly/assembly.xml [hunk 3]
```diff
@@ -25,7 +25,7 @@
     </formats>
     <fileSets>
         <fileSet>
-            <directory>druid_extensions</directory>
+            <directory>target/druid_extensions</directory>
```
Comment (xvrl) [inline distribution/src/assembly/assembly.xml]: same here

File: distribution/src/assembly/assembly.xml [hunk 4]
```diff
@@ -35,7 +35,7 @@
             <outputDirectory>druid_extensions</outputDirectory>
         </fileSet>
         <fileSet>
-            <directory>hadoop_druid_dependencies</directory>
+            <directory>target/hadoop_druid_dependencies</directory>
```
Comment (xvrl) [inline distribution/src/assembly/assembly.xml]: same here

File: distribution/src/assembly/mysql_assembly.xml [hunk 5]
```diff
@@ -25,7 +25,7 @@
     </formats>
     <fileSets>
         <fileSet>
-            <directory>druid_extensions/mysql-metadata-storage</directory>
+            <directory>target/druid_extensions/mysql-metadata-storage</directory>
```
Comment (xvrl) [inline distribution/src/assembly/mysql_assembly.xml]: ditto

File: distribution/pom.xml (hunk 1 above)
Comment (guobingkun) [inline distribution/pom.xml]: yeah, changed to ```${project.build.directory}```

File: distribution/pom.xml (hunk 2 above)
Comment (guobingkun) [inline distribution/pom.xml]: Done.

File: distribution/src/assembly/assembly.xml (hunk 3 above)
Comment (guobingkun) [inline distribution/src/assembly/assembly.xml]: Done.

File: distribution/src/assembly/assembly.xml (hunk 4 above)
Comment (guobingkun) [inline distribution/src/assembly/assembly.xml]: Done.

File: distribution/src/assembly/mysql_assembly.xml (hunk 5 above)
Comment (guobingkun) [inline distribution/src/assembly/mysql_assembly.xml]: Done.

**Comment by gianm** [comment]
:+1:

**Pull request merged**